sdcm/nemesis.py

```diff
@@ -1152,35 +1153,36 @@ def _destroy_data_and_restart_scylla(self, keyspaces_for_destroy: list = None, s
         self.log.debug("Chosen tables: %s", tables)
 
         # Stop scylla service before deleting sstables to avoid partial deletion of files that are under compaction
-        self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
+        with ignore_raft_topology_cmd_failing():
+            self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
```

Comment:
Here i think better to use Context manager only for operations stop/start

Reply:
why ? 

`@decorate_with_context` that already exists on this function is a natural fit for this, and was design for those cases exactly